tests/Language/AbstractTranslationTestCase.php

```diff
@@ -65,7 +65,7 @@ abstract class AbstractTranslationTestCase extends TestCase
         //        DutchTranslationTest::class              => 'nl',
         //        NorwegianTranslationTest::class          => 'no',
         //        PolishTranslationTest::class             => 'pl',
-        //        PortugueseTranslationTest::class         => 'pt',
+        PortugueseTranslationTest::class => 'pt',
         BrazilianTranslationTest::class => 'pt-BR',
```

Comment:
[See](https://github.com/codeigniter4/shield/actions/runs/4499927937/jobs/7918430472?pr=689)
```suggestion
        BrazilianTranslationTest::class  => 'pt-BR',
```

OR you can run :

```console
composer cs-fix tests/Language/AbstractTranslationTestCase
```

Reply:
Its done, but to be honest this is really annoying since I didn't touch that line in my changes.

Reply:
I understand, but we have to use [CodeIgniter Coding Style](https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/pull_request.md#php-style) for code integrity.